src/IO/read_externals.jl

```diff
@@ -0,0 +1,43 @@
+include(joinpath(TASOPT.__TASOPTroot__, "misc/index.inc"))
```

Comment:
Initial thought was that we won't need these functions if we get rid of the `.jl` input files and the `.mdl` outputs but perhaps worth archiving somewhere?